.github/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changes from release 2024/12 to 2024/13
 1. Enhancement - Added D098 complex - thanks to @RadarController (Chris Globe)
 2. AIRAC (2413) - Added TRPs for East Midlands (EGNX) and updated taxiways - thanks to @BenWalker01 (Ben Walker)
+Added new helipads and removed old perimeter that's no longer part of the airfield site - thanks to @RadarController (Chris Globe)
```

Comment:
```suggestion
# Changes from release 2024/13 to 2025/01
1. AIRAC (2501) - Added helipads to and removed perimeter from Rochester (EGTO) SMR- thanks to @RadarController (Chris Globe)

# Changes from release 2024/12 to 2024/13
1. Enhancement - Added D098 complex - thanks to @RadarController (Chris Globe)
2. AIRAC (2413) - Added TRPs for East Midlands (EGNX) and updated taxiways - thanks to @BenWalker01 (Ben Walker)
```
This will be a 2501 AIRAC issue, so the CL entry should be in that block. Also, for the actual entry, please make sure to follow the standard CL entry format detailed in the Style Guide :)